docs/cargo_features.md

```diff
@@ -78,6 +78,7 @@ The default feature set enables most of the expected features of a game engine,
 |serialize|Enable serialization support through serde|
 |shader_format_glsl|Enable support for shaders in GLSL|
 |shader_format_spirv|Enable support for shaders in SPIR-V|
+|spectral_lighting|Enable support for spectral colors (monochromatic light) in lights|
```

Comment:
Rather than monochromatic light, single wavelength light might be a better description? Idk what the usual terminology is.